src/highlight/__tests__/HighlightAnnotation-test.tsx

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import * as ReactRedux from 'react-redux';
+import { shallow, ShallowWrapper } from 'enzyme';
+import HighlightAnnotation from '../HighlightAnnotation';
+
+jest.mock('react', () => ({
+    ...jest.requireActual('react'),
+    useState: jest.fn(),
+}));
+
+describe('components/highlight/HighlightAnnotation', () => {
```

Comment:
I think we decided to use component name as test description in box-annotations.

Reply:
👍